README.md

```diff
@@ -14,8 +14,10 @@ one.
 Quick Links:
 
 * [How does it work](#how-does-it-work)
-* [Set it up on Cloud Run](#setup)
 * [Try it out (locally)](#try-out)
+* [Set it up on Cloud Run](#setup)
+* [Configuration](#configuration)
+* [Observability & Troubleshooting](#troubleshooting)
```

Comment:
this is getting detailed a bit.
we can use `npx markdown-toc -i README.md` to generate this toc.